Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM python:3.7-alpine
+FROM python:3.8-alpine
```

Comment:
this currently triggers a warning:
```
[2019-10-31 19:53:55 +0000] [1] [INFO] Starting gunicorn 19.6.0
[2019-10-31 19:53:55 +0000] [1] [INFO] Listening at: http://0.0.0.0:80 (1)
[2019-10-31 19:53:55 +0000] [1] [INFO] Using worker: sync
/usr/local/lib/python3.8/os.py:1021: RuntimeWarning: line buffering (buffering=1) isn't supported in binary mode, the default buffer size will be used
  return io.open(fd, *args, **kwargs)
[2019-10-31 19:53:55 +0000] [7] [INFO] Booting worker with pid: 7
[2019-10-31 19:53:55 +0000] [8] [INFO] Booting worker with pid: 8
[2019-10-31 19:53:55 +0000] [9] [INFO] Booting worker with pid: 9
[2019-10-31 19:53:55 +0000] [10] [INFO] Booting worker with pid: 10
```

Reply:
[Gunicorn issue](https://github.com/benoitc/gunicorn/issues/2091#issuecomment-547786011)
I guess it's best for now to keep Python37 in dockerfile and upgrade after they will release 20.0.0